### PR TITLE
Created managment command to setup component basic structure

### PR DESCRIPTION
**Pull request by ah7255703**

### CODE REVIEW

**Review by EmilStenstrom — CHANGES_REQUESTED** [review]
Hi! Thank you for taking on the task of generating components with a management command! I've gone through the code and left (a lot of) comments. Don't be discouraged, I'm just eager to get all the details right in this library! :)

Clever trick to use Django's template library to generate the component file! ⭐ 

Before including this, I'd like to see some tests. Otherwise we might inadvertently break this in the future! 

Let me know if you need any help with this, I'd happily merge this with the changes suggested!

File: django_components/management/commands/createcomponent.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+        if not comp_path.exists():
+            comp_path.mkdir()
+        self.create_file(
+            comp_path.joinpath(comp_path.joinpath(file_name + ".py")), py_file
+        )
+        for f in ["html", "css", "js"]:
+            self.create_file(
+                comp_path.joinpath(comp_path.joinpath(file_name + f".{f}")), ""
+            )
```
Comment (EmilStenstrom) [inline django_components/management/commands/createcomponent.py:42]: The template file references style.css, and script.js. Do you agree that the default files should have those names aswell?

**Comment by ah7255703** [comment]
thanks for the explanation but i think we've  got your attention to such feature i will work on these fixes and thank you for ur kind response

File: django_components/management/commands/createcomponent.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+    def create_file(self, file_path, content: str):
+        with open(file_path, "w") as file:
+            file.write(content)
+
+    def handle(self, component_name: str, **options: Any):
+        camelCase = capfirst(component_name)
+        file_name = get_valid_filename(component_name)
+        context = Context({"name": camelCase, "file_name": file_name})
+        with open(THIS_DIR / "templates" / "{comp_name}.py.txt") as file:
```
[the remark this answers is not in the document]
Comment (ah7255703) [reply django_components/management/commands/createcomponent.py:29]: i thought the template rendered would take care of the file name but i forgot to remove the brackets

**Comment by EmilStenstrom** [comment]
ping @ah7255703 - Did you have time to look at the suggestions above?

**Comment by ah7255703** [comment]
> ping @ah7255703 - Did you have time to look at the suggestions above?

Ok, i currently have no time these days
Soon will be done

**Pull request closed without merging**